dcnm/resource_dcnm_inventory.go

```diff
@@ -595,7 +646,10 @@ func resourceDCNMInventroyUpdate(d *schema.ResourceData, m interface{}) error {
 				configTimeout = configTimeout - 5
 			}
 			if migrate {
-				log.Println("Timeout occurs before going into normal mode. Hence removing it!", ip)
+				diags = append(diags, diag.Diagnostic{
+					Severity: diag.Warning,
+					Detail:   fmt.Sprintf("Timeout occurs before going into normal mode. Hence removing it! %s", ip),
```

Comment:
probably should be "timeout" based on other changes.

Reply:
Done